test/e2e/testutils/tester/tester.go

```diff
@@ -42,65 +58,151 @@ func GetTester(ctx context.Context) *Tester {
 func createTester(ctx context.Context) *Tester {
 	kubeconfig1 := os.Getenv("KUBECONFIG_1")
 	if kubeconfig1 == "" {
-		klog.Error("KUBECONFIG_1 not set")
-		os.Exit(1)
+		klog.Fatal("KUBECONFIG_1 not set")
 	}
 	kubeconfig2 := os.Getenv("KUBECONFIG_2")
 	if kubeconfig2 == "" {
-		klog.Error("KUBECONFIG_2 not set")
-		os.Exit(1)
+		klog.Fatal("KUBECONFIG_2 not set")
 	}
+	kubeconfig3 := os.Getenv("KUBECONFIG_3")
+	if kubeconfig3 == "" {
+		klog.Fatal("KUBECONFIG_3 not set")
+	}
+	kubeconfig4 := os.Getenv("KUBECONFIG_4")
+	if kubeconfig4 == "" {
+		klog.Fatal("KUBECONFIG_4 not set")
+	}
```

Comment:
IIUC (@palexster please double check), some tests may still be executed with two clusters only, so this should be configurable (and possibly code duplication should be reduced)

Reply:
Totally agree.